donjo-app/controllers/Penduduk.php

```diff
@@ -532,72 +575,70 @@ public function statistik($tipe = 0, $nomor = 0, $sex = NULL)
 		$this->clear_session();
 		// Untuk tautan TOTAL di laporan statistik, di mana arg-2 = sex dan arg-3 kosong
 		// kecuali untuk laporan wajib KTP
-		if ($sex == NULL AND $tipe <> 18)
+		if ($sex == NULL && $tipe <> 18)
```

Comment:
Saya lebih senang pakai AND dan OR, yg lebih mudah dimengerti. Tapi silakan tambahkan di style guide OpenSID supaya kita konsisten dan tidak hanya mengubah sesuai selera setiap kali.

Reply:
Ini sy ikuti di style guide CI3

https://codeigniter.com/userguide3/general/styleguide.html#class-and-method-naming

```
if ($foo OR $bar)
if ($foo && $bar) // recommended
if ( ! $foo)
if ( ! is_array($foo))
```